src/test/java/lotto/model/WinningLottoTest.java

```diff
@@ -0,0 +1,39 @@
+package lotto.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+class WinningLottoTest {
+    @Test
+    @DisplayName("로또 당첨 번호가 보너스볼의 번호를 포함하면 예외가 발생한다.")
+    void createWrongWinningLotto() {
+        assertThatIllegalArgumentException().isThrownBy(() -> new WinningLotto(new Lotto(1, 2, 3, 4, 5, 6), new LottoNumber(1)));
```

Comment:
```suggestion
        assertThatIllegalArgumentException().isThrownBy(() -> new WinningLotto(new Lotto(1, 2, 3, 4, 5, 6), 1));
```
위와 같이 생성하도록 지원하면 어떨까?